client_integration_test.go

```diff
@@ -0,0 +1,206 @@
+package notify_test
+
+import (
+	"os"
+	"testing"
+
+	notify "github.com/govau/notify-client-go"
+)
+
+func setup(t *testing.T) (client *notify.Client, smsTemplateID string, emailTemplateID string) {
+	apiKey := os.Getenv("API_KEY")
+	if apiKey == "" {
+		t.Fatal("could not set API_KEY environment variable")
+	}
+
+	smsTemplateID = os.Getenv("SMS_TEMPLATE_ID")
+	if smsTemplateID == "" {
+		t.Fatal("could not set SMS_TEMPLATE_ID environment variable")
+	}
+
+	emailTemplateID = os.Getenv("EMAIL_TEMPLATE_ID")
+	if emailTemplateID == "" {
+		t.Fatal("could not set EMAIL_TEMPLATE_ID environment variable")
+	}
+
+	var err error
+	client, err = notify.NewClient(
+		apiKey,
+	)
+	if err != nil {
+		t.Fatal("could not create Notify client", err)
+	}
+	return
+}
+
+func TestSendSMS(t *testing.T) {
+	client, smsTemplateID, _ := setup(t)
```

Comment:
@andrewoh531 see https://github.com/govau/cg-dashboard/commit/7989cc94f18bb85789e9b5f6002973389b9e7336

I think we should consider doing a similar thing e.g. `SKIP_INTEGRATION_TESTS`. Reason being that I checked this out locally and didn't have any env vars set up to successfully run all these tests. Something to consider, I think it might be a commonly used pattern.

Reply:
In the scenario you described, shouldn't you set the env vars up rather than skip the integration tests? If anyone's changing this client, I would them to run the integration tests.

Reply:
Having said that, we currently don't have anywhere to store/share the API Keys securely for the client libraries amongst ourselves and future developers.

Reply:
@andrewoh531 I think it's fine to not do skip for now. Just something to revisit and see if we should do it.